lib/omniauth/strategies/realme.rb

```diff
@@ -29,13 +31,26 @@ def callback_phase
           }
         end
 
-        @raw_info = response
```

Comment:
Doesn't appear to be used or defined anywhere in `OmniAuth` itself. The google-oauth2 strategy uses this variable name extensively so I'm guessing this just a leftover from initial development work